developer/src/tike/main/UfrmMain.pas

```diff
@@ -761,31 +760,6 @@ procedure TfrmKeymanDeveloper.WMUserInputLangChange(var Message: TMessage);
     PostMessage(FChildWindows[i].Handle, WM_USER_INPUTLANGCHANGE, Message.wParam, Message.LParam);
 end;
 
-procedure TfrmKeymanDeveloper.WMUserLoadRegFiles(var Message: TMessage);
```

Comment:
Only used by TikeActive call to load files into existing instance... so no longer used.